ModelicaTest/MultiBody.mo

```diff
@@ -3699,7 +3795,11 @@ a linear damper is connected here.
       extends Modelica.Icons.Example;
       inner MultiBody.World world
         annotation (Placement(transformation(extent={{-96,40},{-76,60}})));
-      parameter Real tol=1e-4;
+      parameter SI.Position rtol=1e-4;
+      parameter SI.Velocity vtol=1e-4;
+
+      final parameter SI.Position s_start = -1 "Initial relative distance between world frame and body";
```

Comment:
May I ask why the parameter is made final, but not making the modifications using it final?

Reply:
No particular reason for this. I just didn't touch the modifiers. The parameter itself is final just to retain its value.